src/components/HTTPHeader.jsx

```diff
@@ -15,9 +16,24 @@ const mappings = {
   },
 };
 
+const lightModeMappings = {
+  POST: {
+    color: "#317854",
+    background: "#e4f5ea",
+    border: "#c1e3d0",
+  },
+  GET: {
+    color: "#4a4aa5",
+    background: "#e8e8fb",
+    border: "#c9c9ee",
+  },
+};
+
+const getStyle = (method, colorMode) => (colorMode === "dark" ? darkModeMappings : lightModeMappings)[method];
+
 const HeaderWrapper = styled.div`
-  border: 1px solid ${(props) => mappings[props.method].border};
```

Comment:
Instead of using react, I would highly recommend using CSS variables which can be set in custom.css

e.g.

```css
:root {
  --header-border-style: 1px solid #c1e3d0;
}

[data-theme="dark"] {
  --header-border-style: 1px solid #c9c9ee;
}
```

Reply:
[Resolved](https://github.com/top-gg/docs/pull/101/commits/bc8dc251ca85c266ba2cb38522ff26c4fb38b5b2). Already tested this one, should be identical.

<img width="3823" height="905" alt="image" src="https://github.com/user-attachments/assets/2c6c1a5f-77a9-4675-8017-77c77ea71a73" />